deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -457,7 +457,7 @@ <h4 class="panel-title pull-left pointer-cursor">
                 currentCell: currentCluster.cellName,
                 hostTypeOptions: info.hostTypes.map(function(item, idx) {
                     return {
-                        value: item.abstract_name,
+                        value: item.id,
                         text: item.abstract_name + " (" + item.core + " cores, " + item.mem + " GB, " + item.storage + ", " + item.provider + ": " + item.provider_name + ")",
                         isSelected: item.abstract_name === currentCluster.hostType
```

Comment:
since we are at it can we change this text to a format:
<Provider_name> (Description: <Abstract_name>, Spec: <8 cores etc...>)

this applied to all 3 files.

Reply:
@yongwen what do you think ?

Reply:
ah, makes sense.